devlib/instrument/gem5power.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, target, power_sites):
             self.power_sites = power_sites
         else:
             self.power_sites = [power_sites]
-        self.add_channel('sim_seconds', 'time')
+        self.add_channel('timestamp', 'time')
```

Comment:
This breaks the `Gem5PowerInstrument` since `sim_second` must be extracted from the statistics file of gem5.